app/Http/Controllers/Api/Auth/OAuthController.php

```diff
@@ -68,6 +67,30 @@ public function login(Request $request)
         }
     }
 
+    /**
+     * Validate the request.
+     *
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse|true
+     */
+    private function validateRequest(Request $request)
+    {
+        $validator = Validator::make($request->all(), ['email' => 'email|required', 'password' => 'required',]);
```

Comment:
Please indent like this:
```suggestion
        $validator = Validator::make($request->all(), [
            'email' => 'email|required',
            'password' => 'required',
        ]);
```